playbooks/libvirt/openshift-cluster/tasks/configure_libvirt_network.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Test if libvirt network for openshift already exists
+  command: "virsh -c {{ libvirt_uri }} net-info {{ libvirt_network }}"
+  register: net_info_result
+  changed_when: False
+  failed_when: "net_info_result.rc != 0 and 'error: Network not found:' not in net_info_result.stderr"
+
+- name: Create a temp directory for the template xml file
+  command: "/usr/bin/mktemp -d /tmp/openshift-ansible-XXXXXXX"
```

Comment:
I have the feeling that you took care of not using mktemp’s absolute path everywhere else in order to avoid issue #154 . It looks like you missed this one.  ;-)